crypto/err/openssl.txt

```diff
@@ -249,6 +250,10 @@ CMS_F_CMS_ADD1_SIGNER:102:CMS_add1_signer
 CMS_F_CMS_ADD1_SIGNINGTIME:103:cms_add1_signingTime
 CMS_F_CMS_ADD1_SIGNING_CERT:181:cms_add1_signing_cert
 CMS_F_CMS_ADD1_SIGNING_CERT_V2:182:cms_add1_signing_cert_v2
+CMS_F_CMS_AUTHENVELOPEDDATA_CREATE:184:
```

Comment:
The CMS_F values are no longer used in 3.0. So you can do CMSErr(0, r) now.
I guess this will make it slightly easier to backport to informal branches of 1.1.1